docs/api.rst

```diff
@@ -8969,6 +8968,7 @@ Adding EVM accounts to all EVM chains
    :resjson object existed: A mapping containing the evm chain keys and which addresses were already tracked before the api call and no action was taken on them.
    :resjson object failed: A mapping containing the evm chain keys and which addresses failed to get added for each chain due to some error contacting remote APIs.
    :resjson object no_activity: A mapping containing the evm chain keys and which addresses don't have activity in that chain.
+   :resjson list no_activity: A list of the addresses that were detected as ethereum contracts.
```

Comment:
you need to change the description of all the 4 mapings above though. The chains are no longer the key. Ping me in discord for suggestion before committing.